ynr/apps/people/management/commands/remove_inactive_person_links.py

```diff
@@ -0,0 +1,49 @@
+import requests
+from django.core.management.base import BaseCommand
+from people.models import Person
+
+
+class Command(BaseCommand):
+    """
+    Test and remove inactive or dead links from Person objects.
+    """
+
+    def handle(self, *args, **options):
+        """
+        Iterate over all Person objects and check if the
+        person identifier urls return a 200 status code.
+        """
+        inactive_links = []
+        people = Person.objects.all()
+        for person in people:
+            person_identifiers = person.get_all_identifiers
+            person_identifiers = [
+                identifier
+                for identifier in person_identifiers
+                if identifier.value.startswith("http")
```

Comment:
We can't detect working Facebook links as sometime they give a 404 even when not logged in. I think we should exclude any Facebook URL (including I think they have `fb.com` or something?)